actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -205,9 +205,10 @@ pub fn call_generic<RT: Runtime>(
                 .map_err(StatusCode::from)
             {
                 Ok(return_data) => (1, return_data),
-                Err(status) => {
-                    let msg = format!("{}", status);
-                    (0, msg.as_bytes().to_vec())
+                Err(_status) => {
+                    // TODO we don't return value which sucks for debugging but we cant have that be used as return data
+                    // let msg = format!("{}", status);
```

Comment:
What's the action here? Is there an issue? I.e., if someone sees this TODO and wants to do something, what should they do?

(I'd also delete the commented code, that'll just get stale)

Reply:
(it would also be nice to move changes to a separate PR from the tests)

Reply:
yeh, got sidetracked by logging things. I've removed this entirely from this PR like you've said and will file an issue for a sec. I wanted to get https://github.com/filecoin-project/builtin-actors/pull/985 out beforehand though since that will be the only way to get meaningful errors from precompiles with this change. 

I probably should've marked this as draft after the first review since I decided to get very hacky with this branch...

Almost done cleaning it up though :)

Reply:
(issue mentioned https://github.com/filecoin-project/ref-fvm/issues/1361)